.github/workflows/pr.yaml

```diff
@@ -15,7 +15,7 @@ jobs:
       name: microovn
       path: microovn
   test:
-    runs-on: ubuntu-latest
+    runs-on: [ ubuntu-latest, large ]
```

Comment:
I'm not sure we have access to large GitHub hosted runners, if we need more resources we could potentially use Canonical self-hosted runners.

Depending on the answer to the comment/question on the `Prepare Environment for functional tests` step, we might get away with a smaller footprint that could fit in regular runner?